e2e/_suites/fleet/ingest_manager_test.go

```diff
@@ -32,31 +32,43 @@ func setUpSuite() {
 	}
 
 	// check if base version is an alias
-	agentVersionBase = e2e.GetElasticArtifactVersion(agentVersionBase)
+	v, err := e2e.GetElasticArtifactVersion(agentVersionBase)
```

Comment:
I'm confused by this line. Where is `agentVersionBase` coming from now that you've deleted it from https://github.com/elastic/e2e-testing/pull/1016/files#diff-3e08fc63f4ba9deffcc23be1d3bdfd0ef89646940c34dd47d06ab797ab6fa673L35 ? Am I missing something obvious?

Reply:
The `agentVersionBase` variable is initialised in https://github.com/elastic/e2e-testing/blob/07f6c4d1c44c64ca15a9f18431a969cf8675ced5/e2e/_suites/fleet/world.go#L29. We have a `world.go` file with the initial values for all constants, and during setup we potentially override them with the env vars provided by CI or the developer.

Reply:
Ah, so it's a global. Interesting. Thanks!

Reply:
@adam-stokes this is something we can improve for the future: how initial configs are created. I know [viper](https://github.com/spf13/viper) has support for initialising the project reading from the environment (and many others). Maybe this is something we can explore to simplify setting up the project. Thoughts?